packages/helpers/packages/ProxyHelpers/index.js

```diff
@@ -1,3 +1,12 @@
+/**
+ * @name onChangeReflector
+ * @description
+ * returns a proxy wrapper to get target safely,
+ * and call onChange callback after defining or deleting any property in created proxy to use in reflector
+ * @param object
+ * @param onChange
+ * @return {boolean|any}
+ */
```

Comment:
Is this syntax for jsDoc?

Reply:
to highlight the name and description of a function you should use the name and description tags
https://jsdoc.app/tags-function.html
https://jsdoc.app/tags-name.html
https://jsdoc.app/tags-descriotion.html

Reply:
Actually I myself do not like jsDoc comments(they made code dirty), by default the first line of comments on function used as description and the name is same. By the way popular libraries and frameworks did not used jsDoc , it is an old school way to document things

Reply:
well, I don't see any dirt on code by commenting this way, besides that, by doing this there is no need to manually write the documents for the helpers, so by running the jsDoc command, the entire helpers would be documented.

Reply:
here is the difference when you don't use the name tag, it detect the function as a constant!
![Screenshot_2020-03-22 JSDoc Global](https://user-images.githubusercontent.com/29011692/77245531-d07e5800-6c3c-11ea-981a-f7b9345250e4.png)
![Screenshot_2020-03-22 JSDoc Global(1)](https://user-images.githubusercontent.com/29011692/77245533-d411df00-6c3c-11ea-8a93-ebf052abc4ff.png)

Reply:
Hmm, well , it seems good to document API for hooks or helpers.
Actually docz has an in progress task to implement jsdoc with docz , but for now we could use jsdoc command

Reply:
oh great, what I supposed to do was to use the json export of the jsDoc, then navigate through the items, per each scop and build an mdx file with menu to let the docz show it.